packages/odyssey-react-mui/src/labs/SideNav/SideNavHeader.tsx

```diff
@@ -11,24 +11,30 @@
  */
 
 import styled from "@emotion/styled";
-import { memo, useMemo, type ReactElement } from "react";
+import { memo } from "react";
+import { Skeleton } from "@mui/material";
+
+import { Box } from "../../Box";
 import {
   type DesignTokens,
   useOdysseyDesignTokens,
 } from "../../OdysseyDesignTokensContext";
-import { Box } from "../../Box";
+import { SideNavLogo } from "./SideNavLogo";
+import { SideNavProps } from "./types";
 import { Heading6 } from "../../Typography";
 import { TOP_NAV_HEIGHT } from "../TopNav";
 
 const SideNavLogoContainer = styled("div", {
   shouldForwardProp: (prop) => prop !== "odysseyDesignTokens",
 })(({ odysseyDesignTokens }: { odysseyDesignTokens: DesignTokens }) => ({
   height: TOP_NAV_HEIGHT,
-  padding: odysseyDesignTokens.Spacing3,
-  borderColor: odysseyDesignTokens.HueNeutral50,
-  borderStyle: odysseyDesignTokens.BorderStyleMain,
-  borderWidth: 0,
-  borderBottomWidth: odysseyDesignTokens.BorderWidthMain,
+  padding: odysseyDesignTokens.Spacing4,
+
+  "svg, img": {
+    height: "100%",
+    width: "auto",
+    maxWidth: "100%",
+  },
```

Comment:
We only need this if they're passing those to us and we're not rendering them ourselves.

Since we're rendering the images now, we should be able to put the styles there ourselves. I think the issue is we can't pass props to `svg` elements in Odyssey.